bin/cli.js

```diff
@@ -4,22 +4,53 @@
 /**
  * Dependencies
  */
+const path = require('path');
 const chalk = require('chalk');
 const argv = require('yargs').argv;
 const replace = require('../lib/replace-in-file');
 
 //Verify arguments
-if (argv._.length < 3) {
+if (argv._.length < 3 && !argv.config) {
   console.error(chalk.red('Replace in file needs at least 3 arguments'));
   process.exit(1);
 }
 
-//Collect main arguments
-let from = argv._.shift();
-const to = argv._.shift();
+let from;
+let to;
+let files;
+
+// If --config is set, load config file
+if (argv.config) {
+  //Read config file
+  let config;
+  try {
+    config = require(path.join(process.cwd(), argv.config));
+  }
+  catch (e) {
+    console.error(chalk.red('Cannot load config file'));
```

Comment:
Probably need to add a `console.error(chalk.red(e));` to notify the user of the error